index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+          <li>
+Return <var>dataModelVerified</var>.
+          </li>
+        </ol>
+
+      </section>
+
+      <section class="normative">
+        <h3>Verify Securing Mechanism</h3>
+
+        <p>
+The algorithm in this section can be used to verify whether a securing mechanism
+that is associated with a sequence of bytes is valid. This algorithm takes a
+byte sequence (bytes <var>inputBytes</var>) and a media type
+(string <var>inputMediaType</var>) as inputs, and produces an object
+(<var>securingMechanismVerified</var>) that contains the following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
+          </li>
+          <li>
+a media type (string <var>mediaType</var>)
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The securing mechanism verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>securingMechanismVerified</var> to an empty object and initialize the
+following fields: <var>status</var> to `false`, <var>warnings</var> to an
+empty array, and <var>errors</var> to an empty array.
+          </li>
+          <li>
+Determine the specific mechanism for extracting the <a>conforming document</a>
+(<var>extractDocument</var>):
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the <var>extractDocument</var> function takes
+<var>inputBytes</var> as an input parameter, parses the value as JSON into
+an object, removes the `proof` property, and returns the result as an object.
+If JSON parsing fails, a <a href="#PARSING_ERROR">PARSING_ERROR</a> is appended
+to <var>securingMechanismVerified</var>.<var>errors</var>.
+              </li>
+              <li>
+If the <var>inputMediaType</var> is associated with the JOSE or COSE
+specifications, the [[?VC-JOSE-COSE]] specification defines a process for
+extracting the <a>conforming document</a>.
+<div class="issue" title="Document extraction process in vc-jose-cose">
+The process for extracting a conforming document is not clearly documented in
+the [[?VC-JOSE-COSE]] specification yet, but once it is, a link to that section
+of the specification will be included in this text.
+</div>
+              </li>
+              <li>
+Other extracting functions are possible, but need not be listed in
+the <a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+Implementers might find these other specifications through other standards
+bodies or communities that utilize this specification.
+              </li>
```

Comment:
```suggestion
```

Removing reference to note from normative section, also not required to be understood

Reply:
@jyasskin, if we remove this text, will it lead to an objection from you?